### DISCUSSION TITLE
请问Clash Verge 是否有纯血鸿蒙的版本

### DISCUSSION THREAD

**Discussion by ttcath** [clash-verge-rev/clash-verge-rev#1897] [General]
请问Clash Verge 是否有纯血鸿蒙的版本，如果有，如何安装。谢谢！

**Comment by tuopgry** [comment]
想自己送上门？

**Reply by ttcath** [reply]
不是，我只是想问问

**Comment by iwalke** [comment]
正常应用都没适配清楚呢。内核那边有个讨论是真的，去看看吧

**Comment by llhdxnq** [comment]
Clash鸿蒙版，我记得上次逛B站，我还看见有开发者开发了

**Reply by jerry-rs** [reply]
> Clash鸿蒙版，我记得上次逛B站，我还看见有开发者开发了

Clash鸿蒙版开发的好，监狱进的早<~.~>

**Reply by Rangtian** [reply]
> > Clash鸿蒙版，我记得上次逛B站，我还看见有开发者开发了
> 
> Clash鸿蒙版开发的好，监狱进的早<~.~>

鸿蒙开发：从入门到入狱

**Comment by cy94120** [comment]
???哥们你是今天第一个让我笑的

**Comment by Dragon1573** [comment]
请考虑使用 [xiaobaigroup/ClashBox](https://github.com/xiaobaigroup/ClashBox) ，或借助 卓易通/出境易 使用 [MetaCubeX/ClashMetaForAndroid](https://github.com/MetaCubeX/ClashMetaForAndroid) 。
